odml/tools/rdf_converter.py

```diff
@@ -19,6 +19,14 @@ class RDFWriter(object):
     Creates the RDF graph storing information about the odML document 
     """
 
+    # Dictionary for mapping known subclasses of a section to ontology namespace
+    section_subclasses = {
```

Comment:
- Make keys all lower case.
- Provide all currently available subclasses.
- The mapping dictionary should be easily update-able w/o touching the converter. Maybe load the list of known subclasses with the namespace mapping from e.g. a provided yaml file (could be done in a later step).